### ISSUE TITLE
Dragonshard looks really weird

### ISSUE THREAD

**Issue by Metalhead33** [doitsujin/dxvk#1899]
The game looks really weird with DXVK being used:

[https://imgbox.com/gallery/edit/BbFvVnCdQc/2vtkeSYN6FwCQdfT](https://imgbox.com/gallery/edit/BbFvVnCdQc/2vtkeSYN6FwCQdfT)

When I use DgVoodoo2 in combination with DXVK, this problem does not exist, but instead all unit textures turn black. But with DgVoodoo2 removed, things become ugly, as demonstrated on the pictures.

### Software information

[Dungeons & Dragons: Dragonshard](https://en.wikipedia.org/wiki/Dungeons_%26_Dragons:_Dragonshard)

### System information

* Linux localhost.localdomain 5.9.11-gentoo #1 SMP x86_64 Intel(R) Core(TM) i3-6100 CPU @ 3.70GHz GenuineIntel GNU/Linux
* GeForce GTX 960/PCIe/SSE2 NVIDIA 455.38
* Wine version: lutris-6.0-rc1-x86_64
* dxvk-1.7.3

### Apitrace file

[https://drive.google.com/file/d/14symu8TUaFrpGUT0pKcZ41-UKPm4T2C2/view?usp=sharing](https://drive.google.com/file/d/14symu8TUaFrpGUT0pKcZ41-UKPm4T2C2/view?usp=sharing)

### Log files
- [d3d9.log](https://drive.google.com/file/d/1n2dN0ZJGmUnaUHKj1MCs5ye5iaTfTc32/view?usp=sharing)

#### DXVK 1.7.3 + DgVoodoo v2.72:

![image](https://cdn.discordapp.com/attachments/561359616135397396/799719598341357589/2021-01-15-201944-screenshot.png)
![image](https://cdn.discordapp.com/attachments/561359616135397396/799719603236110346/2021-01-15-201945-screenshot.png)

#### Just DXVK 1.7.3:

![image](https://cdn.discordapp.com/attachments/561359616135397396/799719956567687218/2021-01-15-202113-screenshot.png)
![image](https://cdn.discordapp.com/attachments/561359616135397396/799719961420759100/2021-01-15-202110-screenshot.png)

In conclusion:

**DXVK + DgVoodoo:** Bridge looks normal, but units are completely black :frog: 
**Just DXVK:** Units have normal textures, but units and bridges alike have weird screwed-up transparency :frog: 
**OpenGL renderer:** Let's found out!

**Comment by Metalhead33** [comment]
The game looks fine with DXVK 1.7.3 and [DgVoodoo 2.64](http://dege.freeweb.hu/dgVoodoo2/dgVoodoo2/)together. The black unit bug is found in newer versions of DgVoodoo 2.

![image](https://cdn.discordapp.com/attachments/561359616135397396/799727163049246760/2021-01-15-204940-screenshot.png)

**Comment by WinterSnowfall** [comment]
I've known about this for a while now and even wanted to report it at one point, but since the game looks all good on an intel iGPU, I assumed the issues were caused by the Nvidia drivers - which I still believe is the case, but maybe something can be done to work around the weird transparency bugs.

**Comment by Metalhead33** [comment]
> I've known about this for a while now and even wanted to report it at one point, but since the game looks all good on an intel iGPU, I assumed the issues were caused by the Nvidia drivers - which I still believe is the case, but maybe something can be done to work around the weird transparency bugs (since dgVoodoo manages to pull that off somehow).
> 
> P.S. The same issues are present even in WineD3D when on an Nvidia GPU.

For the time being, I suggest using DgVoodoo 2.64, as that works fine by me - newer versions of DgVoodoo introduce the bug with black-textured units, while lack of DgVoodoo results in the aforementioned bug.

**Comment by Blisto91** [comment]
Can confirm this issue on Nvidia with latest driver and master dxvk.
Not present on my amd card with mesa.

Did not find any conf options that got rid of it.

**Comment by Metalhead33** [comment]
> Can confirm this issue on Nvidia with latest driver and master dxvk. Not present on my amd card with mesa.
> 
> Did not find any conf options that got rid of it.

The only solution - for an Nvidia card - is to use DgVoodoo 2.64.

Didn't try with any newer version of DgVoodoo after 2011 though.

**Comment by Blisto91** [comment]
Ye but if it can work through  DgVoodoo I'd imagine it should be possible through dxvk alone

**Comment by doitsujin** [comment]
Should be fixed now.
